### DISCUSSION TITLE
Is there a way to retrieve the admin link if I didn't save it at the start?

### DISCUSSION THREAD

**Discussion by laurabrenskelle** [FacilMap/facilmap#283] [Q&A]
I only have the editable link because that is what I shared, but I didn't save the admin link to make changes to the settings. Is there any way I can retrieve that now?

**Comment by cdauth** [comment]
Right now there is no way. What you can do is export your map as GeoJSON and then create a new map where you import all the markers/lines/types/views from the GeoJSON file. I'm hoping to implement an easier copy map function at some point, but for lost admin links I don't really have a very good solution right now. (Maybe at some point I need to introduce user accounts where the recently visited maps are persisted.)

**Comment by gitouttatown** [comment]
I made the same mistake. I don't suppose a feature has been added to retrieve the admin link?
